includes/settings/settings-ppec.php

```diff
@@ -45,9 +45,15 @@
 
 $credit_enabled_label = __( 'Enable PayPal Credit', 'woocommerce-gateway-paypal-express-checkout' );
 if ( ! wc_gateway_ppec_is_credit_supported() ) {
-	$credit_enabled_label .= '<p><em>' . __( 'This option is disabled. Currently PayPal Credit only available for U.S. merchants.', 'woocommerce-gateway-paypal-express-checkout' ) . '</em></p>';
+	$credit_enabled_label .= '<p><em>' . __( 'This option is disabled. Currently PayPal Credit only available for merchants using USD currency.', 'woocommerce-gateway-paypal-express-checkout' ) . '</em></p>';
 }
 
+$location               = wc_get_base_location();
+$credit_enabled_default = 'US' === $location['country'] ? 'yes' : 'no';
```

Comment:
Why doesn't this logic also use currency?

Reply:
See "Enabled by default for US buyers." in https://developer.paypal.com/docs/integration/direct/express-checkout/integration-jsv4/customize-button/ (also, from discussions with PayPal).

Since we're not giving the option for non-US merchants after all, this amounts to having it always enabled when supported.

Reply:
Whoops, this was out of date anyway. Just force pushed a modification that just sets it to `'yes'` in all cases. (I thought I had done this before opening the PR – sorry about that!)